website_sale_one_step_checkout_delivery/__manifest__.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
```

Comment:
Remove all comments in this file and think about some good values for summary, description, etc.